build/metadata.json

```diff
@@ -115,8 +115,8 @@
 		"methods": [],
 		"events": [],
 		"version": "0.1.0-alpha",
-		"url": "https://staging.ink-remix.blockchain-it.hr/",
```

Comment:
could you revert that change please? the circle ci build will do the job

Reply:
Reverted.